docs/user-guide/kubectl/kubectl_version.md

```diff
@@ -83,7 +82,7 @@ kubectl version
 
 * [kubectl](kubectl.md)	 - kubectl controls the Kubernetes cluster manager
 
-###### Auto generated by spf13/cobra at 2015-08-21 17:18:05.1688832 +0000 UTC
+###### Auto generated by spf13/cobra at 2015-09-03 21:06:22.476464324 +0000 UTC
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
 [![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/user-guide/kubectl/kubectl_version.md?pixel)]()
```

Comment:
what is this ga analytics beacon tracker & badge ?!??  bad-ass.  what (if any) open-source project does it stem from?